runtime/near-vm-runner/src/cache.rs

```diff
@@ -52,7 +59,7 @@ pub fn get_contract_cache_key(
     config: &VMConfig,
 ) -> CryptoHash {
     let _span = tracing::debug_span!(target: "vm", "get_key").entered();
-    let key = ContractCacheKey::Version2 {
+    let key = ContractCacheKey::Version3 {
```

Comment:
@olonho This change invalidated old keys in cache and make it pass the upgradable test. Without this change it fails of deserialization cache failed

Reply:
Also @bowenwang1996